js/ui/grid_core/ui.grid_core.keyboard_navigation.js

```diff
@@ -844,21 +931,21 @@ const KeyboardNavigationController = core.ViewController.inherit({
 
     setupFocusedView: function() {
         if(this.isKeyboardEnabled() && !isDefined(this._focusedView)) {
-            this._focusView();
+            this._focusView(this.getView('rowsView'));
```

Comment:
Why?

Reply:
I thought about implementing navigation between views. That is why I added the **view** parameter to the _focusView method. Since we are not planning to implement navigation in other views, I will remove the **view** parameter.